cmd/awscollector/main.go

```diff
@@ -93,6 +93,10 @@ func main() {
 func buildAndParseFlagSet(featgate *featuregate.Registry) (*flag.FlagSet, error) {
 	flagSet := config.Flags(featgate)
 
+	// TODO: remove after ADOT Collector v0.34.0 is released
+	log.Printf("attn: users of the statsd please refer to " +
```

Comment:
nit: do we want to specify the `stasd receiver` here?

Reply:
Yes, the warning issue linked is about the statsd receiver.

Reply:
Oh I meant the wording here like this
```suggestion
	log.Printf("attn: users of the statsd receiver please refer to " +
```